sdks/python/apache_beam/pipeline.py

```diff
@@ -211,6 +211,8 @@ def __init__(self, runner=None, options=None, argv=None):
         experiments.append('beam_fn_api')
         self._options.view_as(DebugOptions).experiments = experiments
 
+    self.local_tempdir = tempfile.mkdtemp(prefix='beam-pipeline-temp')
```

Comment:
Should this be prefixed by something pipeline-specific?

Reply:
Did you have something in mind? I don't think there's a good name here we can use. The system will of course ensure this directory is unique per pipeline.

Reply:
That's fine then, wasn't sure whether this is a unique directory.